docs/Tutorials/CCE.md

```diff
@@ -90,8 +89,12 @@ following datasets:
 - `DrLapse.dat`
 - `DtLapse.dat`
 
-In this format, each row must start with the time stamp, and the remaining
-values are the complex modes in m-varies-fastest format. That is,
+#### Spherical harmonic modes {#cartesian_spherical_modes}
+
+In this format the spherical harmonic coefficients are stored. We use spherical
+harmonic conventions document by the Spherepack class. Each row must start with
```

Comment:
`documented`. Also, we should probably link to some Spherepack docs here. Either our own or something online.